pkg/component/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -380,7 +381,7 @@ func (c *clusterAutoscaler) computeCommand() []string {
 	gardencorev1beta1.SetDefaults_ClusterAutoscaler(c.config)
 
 	command = append(command,
-		fmt.Sprintf("--expander=%s", *c.config.Expander),
+		fmt.Sprintf("--expander=%s", strings.ReplaceAll(string(*c.config.Expander), " ", "")), //remove any spaces in the string
```

Comment:
See above - I would rather enhance the validation to prevent this instead of working around it here in the business logic